server/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Server code for the application",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "node index.js"
```

Comment:
Why have you removed the test script? It should still run that error when it gets called to tell us that it hasn't been implemented??

Reply:
There was not test script 😄 , I will be setting up test framework in next PR.